distribution/docs/src/main/resources/content/_data/data-intro.adoc

```diff
@@ -52,7 +52,7 @@ For example, an image may have different attributes than a PDF document, so each
 
 Most metacards within the system are created using the default metacard type or a metacard type based on the default type.
 The default metacard type of the system can be programmatically retrieved by calling `ddf.catalog.data.impl.MetacardImpl.BASIC_METACARD`.
-The name of the default `MetacardType` can be retrieved from `ddf.catalog.data.MetacardType.DEFAULT_METACARD_TYPE_NAME`.
+The name of the default `MetacardType` can be retrieved from `ddf.catalog.data.MetacardType.DEFAULT{integrating-prefix}metacard_type_NAME`.
```

Comment:
❓  This seems odd place to put the prefix. Is this correct?

Reply:
yeah, that was overly aggressive `find/replace`. Fixed.